### PR TITLE
Implement scale_colors

### PR DESCRIPTION
**Pull request by ziyaointl**
The Color class imposes an upper bound to colors, which constrains
the maximum intensity of the lights. To offer users more freedom, a
separate scale_colors function was implemented that allows color channels
to go beyond 255.

### CODE REVIEW

File: p5/sketch/util.py [hunk 1]
```diff
@@ -9,3 +9,8 @@ def read_shader(filename):
 
 def ensure_p3d(name):
     assert p5.mode == 'P3D', name + "is only available in P3D renderer"
+
+def scale_colors(r, g, b):
```
Comment (parsoyaarihant) [inline p5/sketch/util.py:13]: Can we use another name for this? Its not clear what scale is going to be applied. And it ignores the alpha channel.

Comment (ziyaointl) [reply p5/sketch/util.py:13]: would `div_by_255` work? I guess we can also call it `unbounded_normalize`.

Comment (parsoyaarihant) [reply p5/sketch/util.py:13]: We can use `div_by_255`

Comment (parsoyaarihant) [reply p5/sketch/util.py:13]: Or add a default attribute `scale_tuple(tuple, scale=255):`

**Pull request merged**